storage-resize-images/extension.yaml

```diff
@@ -210,22 +210,22 @@ params:
     example: max-age=86400
     required: false
 
-  - param: IMAGE_TYPE
-    label: Convert image to preferred type
+  - param: IMAGE_TYPES
+    label: Convert image to preferred types
     description: >
-      The image type you'd like your source image to convert to. The default for this option will 
+      The image types you'd like your source image to convert to. The default for this option will 
       be to keep the original file type.
-    type: select
+    type: multiSelect
     options:
-      - label: jpg
-        value: jpg
-      - label: png
-        value: png
+      - label: jpeg
+        value: jpeg
       - label: webp
         value: webp
+      - label: png
+        value: png
       - label: tiff
         value: tiff
-      - label: Do not convert
-        value: false
-    default: false
-    required: false
\ No newline at end of file
+      - label: raw
+        value: raw
+    default: raw
```

Comment:
This ought to be backwards compatible as implicitly suggested by Jeff's [comment](https://github.com/firebase/extensions/issues/575#issuecomment-767149474). 

Here's how I think that would work:

1. Switch this file back to the way it was except change the type to `multiSelect`. 
2. In the code, maintain the check to see if they want image conversion i.e `const hasImageTypeConfigSet = imageType !== "false";`
3. If the above condition is `true` then split the string like you're doing here: `const imageTypes = new Set(config.imageTypes.split(","));` and loop through each type for resize.

Reply:
I'm not sure this would be possible as we had to introduce a wrapper for every image type to then process each image size.

We do currently default to the raw image type that is provided, this should at least create the original file type.

I'm not sure which bit needs to be backwards compatible. I'd assume going forward it would use the newer implementation?

Reply:
Updated to multiSelect. 

Original type added as a default to replace 'do not convert'.